code/go/internal/validator/folder_item_spec.go

```diff
@@ -9,8 +9,8 @@ import (
 	"io/fs"
 	"regexp"
 
-	ve "github.com/elastic/package-spec/v2/code/go/internal/errors"
 	"github.com/elastic/package-spec/v2/code/go/internal/spectypes"
+	ve "github.com/elastic/package-spec/v2/code/go/pkg/errors"
```

Comment:
Or, given that these errors are usually used in the context of the public `validation` package, and are prefixed by `Validation`, wdyt about adding all these errors and associated logic to the public `validation` package, and remove the `Validation` prefixes?

Or could this produce some dependency loop?

Reply:
If it is moved `errors.go`  with the logic and structs to the `validator` package, `import cycle` errors are raised.

It could be moved the whole `errors` package to the `validator` package. But in that case, we maybe have to keep the Validation prefixes in the structs?

Reply:
Rename package to be `specerrors`